GitCommands/Git/GitRevisionTester.cs

```diff
@@ -75,16 +75,7 @@ public bool AnyLocalFileExists(IEnumerable<GitItemStatus> selectedItemsWithParen
                 return true;
             }
 
-            foreach (var item in items)
-            {
-                string filePath = _fullPathResolver.Resolve(item.Name);
-                if (_fileSystem.File.Exists(filePath))
-                {
-                    return true;
-                }
-            }
-
-            return false;
+            return items.Any(item => _fileSystem.File.Exists(_fullPathResolver.Resolve(item.Name)));
```

Comment:
Keep it as `foreach` it has less overhead.

Reply:
reverted and rebased on master